src/controller/scandataexport/execution.go

```diff
@@ -0,0 +1,191 @@
+package scandataexport
+
+import (
+	"context"
+	"fmt"
+	"github.com/goharbor/harbor/src/jobservice/job"
+	"github.com/goharbor/harbor/src/jobservice/logger"
+	"github.com/goharbor/harbor/src/lib/errors"
+	"github.com/goharbor/harbor/src/lib/log"
+	"github.com/goharbor/harbor/src/lib/orm"
+	q2 "github.com/goharbor/harbor/src/lib/q"
+	"github.com/goharbor/harbor/src/pkg/scan/export"
+	"github.com/goharbor/harbor/src/pkg/systemartifact"
+	"github.com/goharbor/harbor/src/pkg/task"
+	"strings"
+	"time"
+)
+
+func init() {
+	task.SetExecutionSweeperCount(job.ScanDataExport, 50)
+}
+
+var Ctl = NewController()
+
+type Controller interface {
+	Start(ctx context.Context, criteria export.Request) (executionId int64, err error)
+	GetExecution(ctx context.Context, executionId int64) (*export.Execution, error)
+	GetExecutions(ctx context.Context, userName string) ([]*export.Execution, error)
+	GetTask(ctx context.Context, executionId int64) (*task.Task, error)
```

Comment:
one execution only for one task? for other resource, it often use ListTasks().

Reply:
When we had originally discussed the proposal, as far as I remember correctly it was required to have only one execution per task since the CSV API and Job framework was supposed to behave that way. 
Also did you imply to rename the method `GetExecutions` to `ListExecutions`?

Reply:
When we had originally discussed the proposal, as far as I remember correctly it was required to have only one execution per task since the CSV API and Job framework was supposed to behave that way. 
Also did you imply to rename the method `GetExecutions` to `ListExecutions`?

Reply:
Currently keep the names as is since these are internal APIs and not directly exposed to the end user via the Rest API.